Subject: include cstring to use memset

Building with gcc-4.3.2 fails like this:

object.cc: In constructor 'Glib::ConstructParams::ConstructParams(const Glib::Class&, const char*, ...)':
object.cc:110: error: 'memset' was not declared in this scope
make[6]: *** [object.lo] Error 1

Fix it by including the proper header file.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 glib/glibmm/object.cc |    1 +
 1 file changed, 1 insertion(+)

Index: glibmm-2.14.1/glib/glibmm/object.cc
===================================================================
--- glibmm-2.14.1.orig/glib/glibmm/object.cc
+++ glibmm-2.14.1/glib/glibmm/object.cc
@@ -27,6 +27,7 @@
 #include <gobject/gvaluecollector.h>
 
 #include <cstdarg>
+#include <cstring>
 
 //Weak references:
 //I'm not sure what the point of these are apart from being a hacky way out of circular references,
